validate.js

```diff
@@ -242,13 +256,13 @@ Validator.prototype.le = function(l, tip) {
 };
 Validator.prototype.contains = function(s, tip) {
 	if (this.goOn && (!isString(this.value) ||!v.contains(this.value,s))) {
-		this.addError(tip || this.key + " is must contain " + s + ".");
+		this.addError(tip || this.key + " must contain " + s + ".");
```

Comment:
There are a lot of lanuage mistakes here. I didn't change everything but did fix a few where I was working on/around that code directly.

Reply:
you reserve the right to make up dates